includes/admin/class-wc-admin-webhooks.php

```diff
@@ -297,32 +297,10 @@ private static function table_list_output() {
 	/**
 	 * Logs output.
 	 *
-	 * @param WC_Webhook $webhook Webhook instance.
+	 * @param WC_Webhook $webhook Deprecated.
 	 */
-	public static function logs_output( $webhook ) {
-		$current = isset( $_GET['log_page'] ) ? absint( $_GET['log_page'] ) : 1; // WPCS: input var okay, CSRF ok.
-		$args    = array(
-			'post_id' => $webhook->get_id(),
-			'status'  => 'approve',
-			'type'    => 'webhook_delivery',
-			'number'  => 10,
-		);
-
-		if ( 1 < $current ) {
-			$args['offset'] = ( $current - 1 ) * 10;
-		}
-
-		remove_filter( 'comments_clauses', array( 'WC_Comments', 'exclude_webhook_comments' ), 10, 1 );
-
-		$logs = get_comments( $args );
-
-		add_filter( 'comments_clauses', array( 'WC_Comments', 'exclude_webhook_comments' ), 10, 1 );
-
-		if ( $logs ) {
-			include_once( dirname( __FILE__ ) . '/settings/views/html-webhook-logs.php' );
-		} else {
-			echo '<p>' . esc_html__( 'This Webhook has no log yet.', 'woocommerce' ) . '</p>';
-		}
+	public static function logs_output( $webhook = 'deprecated' ) {
+		echo '<p>' . sprintf( __( '<a href="%s">View Webhook Delivery Logs</a>', 'woocommerce' ), esc_url( add_query_arg( 'log_file', wc_get_log_file_name( 'webhooks-delivery' ), admin_url( 'admin.php?page=wc-status&tab=logs' ) ) ) ) . '</p>'; // @codingStandardsIgnoreLine
```

Comment:
This forces the use of file based logged.

What if we added a method to the logger class to return a view URL for a certain log; they it could work with both DB based logs, and file based logs.

Reply:
This function is actually not used anymore since the crud changes, going to deprecate it.